### PR TITLE
IGAPP-402: Rename locales to translations

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by maxammann** [comment]
Sorry for the huge PR, here is a filter for easier reviewing: https://github.com/Integreat/integreat-app/pull/89/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.snap&file-filters%5B%5D=.xcscheme&file-filters%5B%5D=.xml&hide-deleted-files=true&file-filters%5B%5D=.sh

File: docs/whitelabelling.md [hunk 1]
```diff
@@ -98,10 +98,10 @@ Things the AppTeam has to do and therefore probably create issues for.
     * 1242 x 2688
     * 2048 x 2732
 * Add to google search console
-* Translate and add locales
+* Translate and add translations
     * New languages
-    * Override locales
-    * Native locales?
+    * Override translations
+    * Native translations?
```
Comment (klinzo) [inline docs/whitelabelling.md:104]: This line can be removed, outdated :)

File: native/android/.idea/jarRepositories.xml [hunk 2]
```diff
@@ -61,5 +61,10 @@
       <option name="name" value="maven4" />
       <option name="url" value="file:$PROJECT_DIR$/../node_modules/react-native-screens/node_modules/react-native/android" />
     </remote-repository>
+    <remote-repository>
```
Comment (klinzo) [inline native/android/.idea/jarRepositories.xml:64]: Is this accidental/necessary?

File: native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme [hunk 3]
```diff
@@ -26,7 +26,7 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Setup native locales"
-               scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-locales setup IntegreatTestCms --locales &quot;../../locales/locales.json&quot; --destination &quot;Integreat&quot;&#10;">
+               scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-translations setup IntegreatTestCms --locales &quot;../../translations/translations.json&quot; --destination &quot;Integreat&quot;&#10;">
```
Comment (klinzo) [inline native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/integreat-test-cms.xcscheme:29]:
```suggestion
               scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-translations setup IntegreatTestCms --translations &quot;../../translations/translations.json&quot; --destination &quot;Integreat&quot;&#10;">
```

File: native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/malte.xcscheme [hunk 4]
```diff
@@ -26,7 +26,7 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Setup native locales"
-               scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-locales setup Malte --locales &quot;../../locales/locales.json&quot; --destination &quot;Integreat&quot;&#10;">
+               scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-translations setup Malte --locales &quot;../../translations/translations.json&quot; --destination &quot;Integreat&quot;&#10;">
```
Comment (klinzo) [inline native/ios/Integreat.xcodeproj/xcshareddata/xcschemes/malte.xcscheme:29]:
```suggestion
               scriptText = "exec &gt; ${PROJECT_DIR}/prebuild_native_locales.log 2&gt;&amp;1&#10;cd ${PROJECT_DIR} &amp;&amp; ../tools/native-translations setup Malte --translations &quot;../../translations/translations.json&quot; --destination &quot;Integreat&quot;&#10;">
```

File: translations/README.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+# Translations for the Integreat frontend
```
Comment (klinzo) [inline translations/README.md:1]: Did not review this :)

**Review by klinzo — APPROVED** [review]

**Comment by klinzo** [comment]
Did you check if there are still occurrences of the term `locales`?

File: native/android/.idea/jarRepositories.xml (hunk 2 above)
Comment (maxammann) [reply native/android/.idea/jarRepositories.xml:64]: I think this is fine, not exactly sure about this file though

File: README.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ Both projects are whitelabelled with different [build configs](docs/build-config
     
 There are also separate sub-projects used by both projects:
     
-* **[locales](locales/README.md)**: Localized texts and utilities to manage them
+* **[translations](translations/README.md)**: Localized texts and utilities to manage them
```
Comment (Taggotty) [inline README.md:37]:
```suggestion
* **[translations](translations/README.md)**: Translated texts and utilities to manage them
```

File: native/docs/i18n.md [hunk 7]
```diff
@@ -23,9 +23,9 @@ UI elements are:
 * Landing page
 * LastUpdate
 
-### Locales
+### Translations
 
-The localized texts (locales) can be found in the [locales sub-project](../../locales). For more information see its [README](../../locales/README.md).
+The localized texts (translations) can be found in the [translations sub-project](../../translations). For more information see its [README](../../translations/README.md).
```
Comment (Taggotty) [inline native/docs/i18n.md:28]:
```suggestion
The translated texts (translations) can be found in the [translations sub-project](../../translations). For more information see its [README](../../translations/README.md).
```

File: translations/README.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+## Content
+* [Submitting for translation](#submitting-for-translation)
+* [Conversion between JSON, CSV and ODS](#conversion-between-json-csv-and-ods)
+* [Used file formats](#used-file-formats)
+
+## Submitting for translation
+
+Translations should be done by professional translators. If there are enough untranslated strings,
+they can be submitted to professional for translations as follows:
```
Comment (Taggotty) [inline translations/README.md:11]:
```suggestion
they can be submitted to professionals for translations as follows:
```

**Review by Taggotty — APPROVED** [review]

**Pull request merged**